src/command/encrypt.iced

```diff
@@ -42,7 +42,7 @@ exports.Command = class Command extends Base
     name = "encrypt"
     sub = scp.addParser name, opts
     add_option_dict sub, @OPTS
-    sub.addArgument [ "them" ], { nargs : 1 , help : "the username of the receiver" }
+    sub.addArgument [ "them" ], { nargs : 1 , help : "the username of the receiver (comma-sep for multiple)" }
```

Comment:
Instead of splitting on commas, let's use the special `nargs : '+'` syntax, which means one or more positional args. That'll get in the way of parsing `file`, so maybe we should relegate that parameter to a command line flag? (That would fit the same pattern as the existing `-m` flag, I think.)